ci/test-conformance-eks.sh

```diff
@@ -173,12 +185,37 @@ function setup_eks() {
     aws --version
 
     set +e
-    aws configure << EOF
-${AWS_ACCESS_KEY}
-${AWS_SECRET_KEY}
-${REGION}
-JSON
+    if [[ "$AWS_SERVICE_USER_ROLE_ARN" != "" ]] && [[ "$AWS_SERVICE_USER_NAME" != "" ]]; then
+        mkdir -p ~/.aws
+        echo > ~/.aws/config <<EOF
+[default]
+region = $REGION
+role_arn = $AWS_SERVICE_USER_ROLE_ARN
+source_profile = $AWS_SERVICE_USER_NAME
+output = JSON
+EOF
+        echo > ~/.aws/credntials <<EOF
```

Comment:
credntials is a typo, can it work?

Reply:
actually i made this change locally when i got error while testing, but didn't uploaded the patch here.